server/handlers/stakeholderHandler.js

```diff
@@ -33,214 +16,202 @@ const stakeholderGetQuerySchema = Joi.object({
   email: Joi.string(),
   phone: Joi.string(),
   website: Joi.string(),
-  children: Joi.array(),
-  parents: Joi.array(),
-  users: Joi.array(),
-  filter: Joi.object(),
+  search: Joi.string(),
+}).unknown(false);
+
+const stakeholderPostSchema = Joi.object({
+  type: Joi.string(),
+  email: Joi.string(),
+  phone: Joi.string(),
+}).unknown(false);
+
+const stakeholderDeleteSchema = Joi.object({
+  id: Joi.string().uuid(),
+  type: Joi.string(),
+}).unknown(false);
+
+const updateStakeholderSchema = Joi.object({
+  id: Joi.string().uuid().required(),
+  email: Joi.string().email(),
+  first_name: Joi.string(),
+  last_name: Joi.string(),
+  logo_url: Joi.string(),
+  map: Joi.string(),
+  org_name: Joi.string(),
+  phone: Joi.string(),
+  website: Joi.string().uri(),
 }).unknown(false);
```

Comment:
As I worked on the client I found that the schemas for deleting and updating were blocking the requests.

Could we update to allow the first_name, last_name, and org_name to have empty string values?  Ultimately, it would be great to be able to require either the org_name OR the first and last_name to have values but right now all the values are being passed along.

These settings seem to work:

const stakeholderDeleteSchema = Joi.object({
  id: Joi.string().uuid(),
  type: Joi.string(),
  linked: Joi.boolean(),
  data: Joi.object({
    id: Joi.string().uuid().required(),
    email: Joi.string().email(),
    first_name: Joi.string().allow(''),
    last_name: Joi.string().allow(''),
    org_name: Joi.string().allow(''),
    logo_url: Joi.string(),
    map: Joi.string(),
    phone: Joi.string(),
    website: Joi.string().uri(),
    children: Joi.array().items(Joi.object()),
    parents: Joi.array().items(Joi.object()),
    image_url: Joi.string().allow(''),
    type: Joi.string(),
    created_at: Joi.string(),
    updated_at: Joi.string(),
  }).unknown(false),
}).unknown(false);

const updateStakeholderSchema = Joi.object({
  id: Joi.string().uuid().required(),
  email: Joi.string().email(),
  first_name: Joi.string().allow(''),
  last_name: Joi.string().allow(''),
  org_name: Joi.string().allow(''),
  logo_url: Joi.string(),
  map: Joi.string(),
  phone: Joi.string(),
  website: Joi.string().uri(),
  children: Joi.array().items(Joi.object()),
  parents: Joi.array().items(Joi.object()),
  image_url: Joi.string().allow(''),
  type: Joi.string(),
}).unknown(false);